src/components/pages/OERInformation.tsx

```diff
@@ -0,0 +1,135 @@
+
+import { useLibraryTranslation } from '../../config/i18n';
+import LIBRARY from '../../langs/constants';
+import VideoWithCC from '../collection/oer/VideoWithCC';
+import DiscoverButton from '../common/DiscoverButton';
+import EmailButton from '../common/EmailButton';
+import FacebookButton from '../common/FacebookButton';
+import TwitterButton from '../common/TwitterButton';
+import MainWrapper from '../layout/MainWrapper';
+import StyledBackgroundContainer from '../layout/StyledBackgroundContainer';
+import StyledBox from '../layout/StyledContainer';
+
+const references = [
+  {
+    href: 'https://www.bpb.de/mediathek/video/234994/oer-erklaert-die-grundlagen/',
+    name: 'OER erklärt – die Grundlagen',
+  },
+  {
+    href: 'https://www.bpb.de/mediathek/video/234998/oer-erklaert-ueber-die-qualitaet-der-materialien/',
+    name: 'OER erklärt – über die Qualität der Materialien',
+  },
+];
+
+const OERInformation = () => {
+  const { t } = useLibraryTranslation();
+
+  return (
+    <StyledBackgroundContainer>
+      <MainWrapper>
+        <Box py={10}>
+          <Typography variant="h1" align="center" color="white">
+            <ImportContacts fontSize="large" sx={{ mr: 1 }} />
+            {t(LIBRARY.OER_INFORMATION_PAGE_TITLE)}
+          </Typography>
+          <Typography variant="subtitle1" align="center" color="white">
+            {t(LIBRARY.OER_INFORMATION_SUBTITLE)}
+          </Typography>
+        </Box>
+
+        {/* share options */}
+        <Box sx={{ background: 'white', textAlign: 'right' }}>
+          <Box sx={{ textAlign: 'right' }} p={2}>
+            <FacebookButton showBorder iconSize="medium" />
+            <TwitterButton
+              showBorder
+              iconSize="medium"
+              message={t(LIBRARY.OER_INFORMATION_PAGE_DESCRIPTION)}
+            />
+            <EmailButton
+              showBorder
+              iconSize="medium"
+              name={t(LIBRARY.OER_INFORMATION_PAGE_TITLE)}
+              description={t(LIBRARY.OER_INFORMATION_PAGE_DESCRIPTION)}
+            />
+          </Box>
+          <Divider variant="middle" />
+        </Box>
+
+        {/* first section */}
+        <StyledBox>
+          <Container>
+            <Typography align="justify" variant="h2">
+              {t(LIBRARY.OER_INFORMATION_BASICS_TITLE)}
+            </Typography>
+            <Typography py={2}>
+              <Trans
+                i18nKey={LIBRARY.OER_INFORMATION_BASICS_TEXT}
+                components={{ bold: <strong /> }}
+              />
+            </Typography>
+
+            <VideoWithCC
+              url="https://www.youtube.com/embed/DfJWqoEc6BI?si=fuhfrbKpqXUN4Vcx"
+              title={t(LIBRARY.OER_INFORMATION_BASICS_TITLE)}
+              production="Kooperative Berlin"
+              duration="3 Min."
+              edition="Bundeszentrale für politische Bildung"
+            />
+          </Container>
+        </StyledBox>
```

Comment:
Same here, extract to a local component ?

Reply:
I think it's good for now, not sure how this page will evolve anyway.